desci_sense/demos/st_demo.py

```diff
@@ -254,6 +258,17 @@ def process_post(post: RefPost, model):
                     wandb_run = init_wandb_run(model.config)
                     log_pred_wandb(wandb_run, result, manual_label, labeler_name)
                     wandb_run.finish()
+                #print(post.dict()), .update({"_id":post.metadata.sourcenetwoork+post.metadata.id}))
```

Comment:
let's add a config section for arango, that can also have a flag for whether to do this part or not? So we should still be able to run the code even without configuring arango

Reply:
the config is here, we can add an arango section to it https://github.com/csensemakers/desci-sense/blob/main/desci_sense/configs.py

Reply:
Ok